.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Package and run all tests
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Init Coveralls
+      shell: bash
+      run: |
+          COVERALLS_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }}
+          if [[ -z "${COVERALLS_TOKEN}" ]];
+          then
+             echo "Coveralls token not available"
+             COVERALLS_SKIP=true
+          else
+             echo "Coveralls token available"
+             COVERALLS_SKIP=false
+          fi
+          echo ::set-env name=COVERALLS_SKIP::${COVERALLS_SKIP}
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Run Maven Targets
+      run: mvn package jacoco:report spotbugs:spotbugs checkstyle:checkstyle coveralls:report --batch-mode --show-version --activate-profiles coveralls -Dcoveralls.skip=$COVERALLS_SKIP -DrepoToken=${{ secrets.COVERALLS_REPO_TOKEN }}
```

Comment:
Ah, I just remembered the `coveralls` profile doesn't exist in the version of the `oss-parent` that is currently in use here. Please upgrade the parent pom to `5.0.0` in this PR and then this will work.